app/jobs/concerns/maintenance_tasks/task_job_concern.rb

```diff
@@ -36,6 +36,8 @@ def build_enumerator(_run, cursor:)
       @enumerator = nil
 
       collection_enum = case collection
+      when :no_collection
+        enumerator_builder.build_once_enumerator(cursor: nil)
```

Comment:
https://github.com/Shopify/job-iteration/blob/53767c94295af5655971c3b33397da692227519f/lib/job-iteration/enumerator_builder.rb#L40

I believe that means that `1` will be passed to the enumerator, if anyone else was curious.

Reply:
> I believe that means that 1 will be passed to the enumerator

Yes! I was also a bit intrigued when I saw this. Indeed, that means that `1` is passed to the user's `process` method, but I don't think it's a big deal either 😄